client-mvp-04/src/components/common/dropdown/dropdown.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+
+const Dropdown = ({ data, setTimezone }) => {
+  const [clickStatus, setClickStatus] = useState(false);
+  const [selectedValue, setSelectedValue] = useState(data.timezones[0]);
+  return (
+    <div className={'dropdown-wrapper'}>
+      <div
+        className={'dropdown'}
+        onClick={() => {
+          setClickStatus(!clickStatus);
+        }}
+      >
+        <div className={'selected'}>{selectedValue}</div>
+        <div className={'arrow'}></div>
+      </div>
+      <div className={`option-container ${!clickStatus ? 'hide' : ''}`}>
```

Comment:
As you don't have `class` for `clickStatus`.
This line could be replaced with:
`<div className={`option-container ${!clickStatus && 'hide'}`}>`